src/windows/guest_list.c

```diff
@@ -179,6 +179,102 @@ void window_guest_list_open()
 }
 
 /**
+* type == 0 -> guests on ride
+* type == 1 -> guests in queue
+* type == 2 -> guests thinking about ride
+* type == 3 -> guests thinking X, opened from news item
+* ebx is number of the ride or index of the thought
+* values of eax and edx probably determine the filter name string
```

Comment:
may as well change ebx to index. Since its a bit more informative than the register name.